docs/install-macos.md

```diff
@@ -0,0 +1,87 @@
+# macOS Install Guide
+
+!!! note "This guide was created using macOS 11.1"
+    It is expected that this guide should work from 10.14 upwards but has only been tested with 11.1
+
+Open Terminal.app by clicking the magnifying glass icon in the top right of your screen.
+Type `terminal` and hit enter.
+
+Install [Homebrew](https://brew.sh/), this should install the Xcode Command Line Tools for you as well.
+
+!!! note "This command might take a while to run depending on the speed of your internet connection."
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Once homebrew is installed use it to install [pipx](https://pipxproject.github.io/pipx/) and [pyenv](https://github.com/pyenv/pyenv):
+
+```bash
+brew install pipx pyenv
+```
+
+Next, install [Docker for Mac](https://docs.docker.com/docker-for-mac/install/), [GitHub Desktop](https://desktop.github.com/), and [Visual Studio Code](https://code.visualstudio.com/):
+
+```bash
+brew install --cask docker github visual-studio-code
+```
+
+Use pyenv to install Python:
+
+!!! note "This command might take a while to run depending on the speed of your computer."
+
+```bash
+pyenv install 3.9.1
+```
+
+Then install the [OpenSAFELY CLI](../opensafely-cli) with pipx, using the Python installed in the previous step:
+
+```bash
+pipx install opensafely --python ~/.pyenv/shims/python3.9
+```
+
+Test the installation of OpenSAFELY CLI.
+This should print out the usage and available sub commands:
+
+```bash
+% opensafely --help                                                                                   12:01:19
```

Comment:
typo: remove `12:01:19`

Reply:
What on… oh I copy pasted this whole block from my terminal didn't I 🤦‍♂️